src/sinks/file/mod.rs

```diff
@@ -18,7 +18,8 @@ use tokio::timer::Delay;
 pub struct FileSinkConfig {
     pub path: Template,
     pub idle_timeout_secs: Option<u64>,
-    pub encoding: Encoding,
+    #[serde(default)]
```

Comment:
Is this missing the skip stuff?

Reply:
I'm not sure if we want it here. 🤔 Hmm... Yeah, I think so. Will update.

Reply:
Just seemed like an odd one out :)